src/components/action-buttons.component.test.tsx

```diff
@@ -0,0 +1,230 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import ActionButtons from "./action-buttons.component";
+import { MedicationRequest } from "../types";
+import { useConfig } from "@openmrs/esm-framework";
+
+const mockedUseConfig = useConfig as jest.Mock;
+
+describe("Action Buttons Component tests", () => {
+  const mockedMutate = jest.fn(() => "mocked mutate");
+
+  beforeEach(() => {
+    mockedUseConfig.mockReturnValue({
+      medicationRequestExpirationPeriodInDays: 90,
```

Comment:
I have a question not related to this test. Do you know if the expiration date configurable via the order template or any where else?

Reply:
Yep, this is a configurable parameter in the dispensing config:

https://github.com/openmrs/openmrs-esm-dispensing-app/blob/main/src/config-schema.ts#L28